tests/integration/test_bibtex_exporting.py

```diff
@@ -22,124 +22,161 @@
+    ], persons={
+        'editor': [],
+        'author': [Person(u"Glashow, S.L.")],
+    }))
+
+    schema = PybtexSchema()
+    result = schema.load(article)
+
+    assert result is not None
+    assert pybtex_entries_equal(result, expected)
 
-    assert expected == result
+
+def test_format_inproceeding(app):
+    inproceedings = get_es_record('lit', 524480)
+    expected = ("Hu:2000az", Entry('inproceedings', [
+        ('key', '524480'),
+        ('address', u"Tokyo, Japan"),
+        ('booktitle', u"4th RESCEU International Symposium on Birth and Evolution of the Universe"),
+        ('title', u"CMB anisotropies: A Decadal survey"),
+        ('archivePrefix', u'arXiv'),
+        ('eprint', u"astro-ph/0002520"),
+        ('primaryClass', u"astro-ph"),
+        ('SLACcitation', u"%%CITATION = ASTRO-PH/0002520;%%"),
+        ('url', u"http://alice.cern.ch/format/showfull?sysnb=2178340"),
+    ], persons={
+        'editor': [],
+        'author': [Person(u"Hu, Wayne")],
+    }))
+
+    schema = PybtexSchema()
+    result = schema.load(inproceedings)
+
+    assert result is not None
+    assert pybtex_entries_equal(result, expected)
 
 
-@pytest.mark.xfail
 def test_format_proceeding(app):
-    proceeding = get_db_record('lit', 701585)
-
-    expected = u'''@proceedings{Alekhin:2005dx,
-      author         = "De Roeck, A. and Jung, H.",
-      title          = "{HERA and the LHC: A Workshop on the implications of HERA for LHC physics: Proceedings Part A}",
-      organization   = "CERN",
-      publisher      = "CERN",
-      address        = "Geneva",
-      url            = "http://weblib.cern.ch/abstract?CERN-2005-014",
-      year           = "2005",
-      pages          = "pp.1-326",
-      eprint         = "hep-ph/0601012",
-      archivePrefix  = "arXiv",
-      primaryClass   = "hep-ph",
-      reportNumber   = "CERN-2005-014, DESY-PROC-2005-01",
-      SLACcitation   = "%%CITATION = HEP-PH/0601012;%%"
-}'''
-    result = Bibtex(proceeding).format()
-
-    assert expected == result
-
-
-@pytest.mark.xfail
-def test_format_thesis(app):
-    thesis = get_db_record('lit', 1395663)
-
-    expected = u'''@phdthesis{Mankuzhiyil:2010jpa,
-      author         = "Mankuzhiyil, Nijil",
-      title          = "{MAGIC $\\gamma$-ray observations of distant AGN and a study of source variability and the extragalactic background light using FERMI and air Cherenkov telescopes}",
-      school         = "Udine U.",
-      year           = "2010",
-      url            = "https://magicold.mpp.mpg.de/publications/theses/NMankuzhiyil.pdf",
-      SLACcitation   = "%%CITATION = INSPIRE-1395663;%%"
-}'''
-    result = Bibtex(thesis).format()
-
-    assert expected == result
-
-
-@pytest.mark.xfail
+    proceedings = get_es_record('lit', 701585)
+    expected = ("Alekhin:2005dx", Entry('proceedings', [
+        ('key', '701585'),
+        ('address', u"Geneva"),
+        ('pages', u"pp.1--326"),
```

Comment:
`pp.` is not needed.

Reply:
'pp.1' is the value of page_start in the publication_info of that record. Should I be stripping it manually then?

Reply:
no, that's something we were doing on legacy, that I forgot about, and that I should discuss with @jacquerie and @jmartinm.